portfolio/src/main/webapp/script.js

```diff
@@ -32,16 +32,34 @@ function funFactGenerator() {
   factGetter.innerText = fact;
 }
 
-function serverFunFact() {
-    
+/**
+ * Displays a fun fact about Desmond on the page.
+ */
+function serverFunFact() { 
     fetch('/data').then(facts => facts.json()).then(jsonfacts => {
 
-    // Pick a random fact.
-    const fact = jsonfacts[Math.floor(Math.random() * jsonfacts.length)];
+        // Pick a random fact.
+        const fact = jsonfacts[Math.floor(Math.random() * jsonfacts.length)];
+
+        // Add it to the page.
+        const factGetter = document.getElementById('factgetter');
+        factGetter.innerText = fact;
+    })
+}
+
+/**
+ * Adds user's comment to the page.
+ */
+function showComments() {
 
-    // Add it to the page.
-    const factGetter = document.getElementById('factgetter');
-    factGetter.innerText = fact;
+    // Starts post request using user's input.
+    var init = {method: "POST",
```

Comment:
you probably don't need this variable. Just create the JSON object when passing in the request argument in the next line.

Reply:
Alright. I'll do that